### PR TITLE
Minor typo fix to main readme

### PR DESCRIPTION
**Pull request by cijothomas**
Fixes #.

## Changes

Please provide a brief description of the changes here.

For significant contributions please make sure you have completed the following items:

* [ ] Appropriate `CHANGELOG.md` updated for non-trivial changes
* [ ] Design discussion issue #

### CODE REVIEW

**Review by mic-max — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -141,7 +141,7 @@ TBD
 
 See [CONTRIBUTING.md](CONTRIBUTING.md)
 
-We meet weekly Monday's at 8:15 PT. The meeting is subject to change depending on
+We meet weekly Monday's at 8:15 AM PT. The meeting is subject to change depending on
```
Comment (reyang) [inline README.md:144]: I don't have strong opinion for this change, I think it is following https://github.com/open-telemetry/community#special-interest-groups.

**Review by reyang — APPROVED** [review]

File: README.md [hunk 2]
```diff
@@ -32,7 +32,7 @@ all the images will be build):
 docker compose up
 ```
 
-- Once the images are build you can access the Webstrore at: <http://localhost:8080>
+- Once the images are build you can access the Webstore at: <http://localhost:8080>
```
Comment (reyang) [inline README.md:35]: "are build" or "are built"?

File: README.md (hunk 1 above)
Comment (cijothomas) [reply README.md:144]: mm.. the client instrument has AM specified!

**Review by julianocosta89 — APPROVED** [review]

**Comment by cartersocha** [comment]
I added the fixes to my [pr](https://github.com/open-telemetry/opentelemetry-demo-webstore/pull/75). We can close this.

**Pull request closed without merging**